### ISSUE TITLE
HE Battle

### ISSUE THREAD

**Issue by damian1231** [shanapu/MyJailbreak-CSS#6]
HE Battle glitches and cant throw HE's after the first couple...

**Comment by shanapu** [comment]
does this happen to everyone everytime?
Can't reproduce...

**Comment by damian1231** [comment]
yeah mate everytime it glitches out but on random amount of grenades i was thinking it was to be unlimited amount?

**Comment by shanapu** [comment]
is unlimited. Just played 2-3 round hebattle against 3 bot without problems. 
I know that sometimes very very rarly players don't get a new grenade - this is a know issue since first release, but not always maybe 1 player in 20 rounds got this problem.
Do you have steps to reproduce it?

**Comment by damian1231** [comment]
tested again 3 times in a row ended up not being able to throw a HE just gets stuck like this and literally cant do anything.
![he](https://user-images.githubusercontent.com/32046008/33246987-9ef124ea-d312-11e7-9ac1-950f9eadd15d.png)
It shows i have a HEGrenade but i cant throw it.

**Comment by shanapu** [comment]
try these, enabled knife with no damage to fix the nade bug

**Comment by damian1231** [comment]
I have tested new version
For some reason players get 85HP to start with?
and knifes do do damage.

The knife kill was done before HE Battle was over
![heba](https://user-images.githubusercontent.com/32046008/33298484-8403a32a-d3de-11e7-8f22-d7ab0b23dd31.png)
